.jshintrc

```diff
@@ -2,6 +2,7 @@
   "predef": [
     "document",
     "window",
+    "moment",
```

Comment:
@jgwhite this looks important but I don't know what it does? can you give me the 10 second version?